src/Xhgui/Searcher/Pdo.php

```diff
@@ -0,0 +1,244 @@
+<?php
+
+class Xhgui_Searcher_Pdo implements Xhgui_Searcher_Interface
+{
+    /**
+     * @var Xhgui_Saver_Pdo
+     */
+    private $saver;
+
+    /**
+     * @var PDO
+     */
+    private $pdo;
+
+    /**
+     * @var string
+     */
+    private $table;
+
+    /**
+     * @param Xhgui_Saver_Pdo $saver
+     * @param PDO             $pdo   An open database connection
+     * @param string          $table Table name where Xhgui profiles are stored
+     */
+    public function __construct(Xhgui_Saver_Pdo $saver, PDO $pdo, $table)
+    {
+        $this->saver = $saver;
+        $this->pdo = $pdo;
+        $this->table = $table;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function latest()
+    {
+        $stmt = $this->pdo->query("
+          SELECT
+            id,
+            profile,
+            url,
+            SERVER,
+            GET,
+            ENV,
+            simple_url,
+            request_ts,
+            request_ts_micro,
+            request_date
+          FROM {$this->table}
+          ORDER BY request_date ASC
+          LIMIT 1
+        ");
+
+        if (false === $row = $stmt->fetch(PDO::FETCH_ASSOC)) {
```

Comment:
Inline assignment and comparisons in the same line is harder to read than using 2 lines.

Reply:
Done https://github.com/perftools/xhgui/pull/244/commits/d801119f3656eccbc6c398024940cdbbd9e42d04